index.html

```diff
@@ -0,0 +1,32 @@
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/styles/app.css">
+    <title>Document</title>
+</head>
+<body class="page">
+    <header class="header">
+        <img src="assets/icons/menu.svg" alt="Menu">
+    </header>
+    <section>
+        <div class="headline"> 
+            <h1 class="lines"> Hey there.</h1>
+            <h1 class="lines"> I'm THIERRY</h1>
+            <h1 class="middle lines">I'm a Software Developer</h1>
```

Comment:
- [ ] You can take these three texts in one `h1`  like this: 
```
<h1 class="lines"> Hey there <br>
           I'm THIERRY <br>
           I'm a Software Developer</h1>
```
The reason is we should not take more than one `h1` in each webpage because we need to be following best semantics and best practices.